war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -728,20 +728,6 @@ function expandButton(e) {
     layoutUpdateCallback.call();
 }
 
-function inputHasDefaultTextOnFocus() {
-    if (this.value == defaultValue) {
-        this.value = "";
-        Element.removeClassName(this, "defaulted");
-    }
-}
-
-function inputHasDefaultTextOnBlur() {
```

Comment:
Ditto

Reply:
not used https://github.com/search?q=org%3Ajenkinsci+inputHasDefaultTextOnBlur&type=Code